etc/rose-meta/rose-suite-info/rose-meta.conf

```diff
@@ -50,6 +41,19 @@ help=User ID of the owner of the suite.
 pattern=^.+(?# Must not be empty)
 sort-key=02-users-1
 
+[=period]
```

Comment:
This is a project specific metadata option, so it shouldn't be in the generic configuration